debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/Replicator.java

```diff
@@ -430,6 +432,7 @@ protected boolean handleOplogEvent(ServerAddress primaryAddress, Document event)
                     logger.error("Get current primary executeBlocking", e);
                 }
                 ServerAddress serverAddress = address.get();
+
```

Comment:
While you're making changes, it'd be great to avoid unrelated changes like this.